kernel/malloc.c

```diff
@@ -230,38 +240,38 @@ static void * f_sbrk(int flags, int incr)
         heap_stack = &_stack - 4096;
     }
 
-    if (flags & MEM_USER) {
+    if ((flags & MEM_USER) == MEM_USER) {
```

Comment:
Hey @ichigo663 can I ask why you prefer it so?

I know this will explicitly transform it in a boolean, but at the end, I see it less clean than as it was before..